### ISSUE TITLE
.route file inconsistent with vpr.out report

### ISSUE THREAD

**Issue by kmurray** [verilog-to-routing/vtr-verilog-to-routing#81]
Originally reported on Google Code with ID 88

```
What steps will reproduce the problem?
1. Run the VTR flow.
This should work with different arch files and benchmarks, but for simplicity, assume
the basic_flow:
arch: timing/k6_frac_N10_mem32K_40nm.xml
verilog: ch_intrinsics.v

What is the expected output? What do you see instead?
vpr.out will report a number of segments used.
In the case above: "Total wiring segments used: 1360"
Counting the CHANX and CHANY lines in the .route file (without duplicates) gives a
different number: 1273

What version of the product are you using? On what operating system?
V7.0, on Ubuntu 12.04.4 LTS

Please provide any additional information below.
I tried to debug this on my own but ended up really confused.
See attached modified version of vpr/route/route_common.c

There are three comments showing where I modified things: search for MOD1, MOD2, MOD3

MOD1 is just a modified version of the function
vpr/SRC/base/stats.c:get_num_bends_and_length()

Basically the modified route_common.c computes segments stats again, and prints the
results both to vpr.out and the .route file. You'll notice that the two consecutive
print statements (MOD2 and MOD3), with the same arguments, give different values in
both files, and I have no idea why this is happening...

```

Reported by `edin.kadric.iv` on 2014-06-28 13:46:04

<hr>
- _Attachment: [route_common.c](https://storage.googleapis.com/google-code-attachments/vtr-verilog-to-routing/issue-88/comment-0/route_common.c)_
